tools/docs/build.bash

```diff
@@ -40,17 +40,22 @@ rm -f docs/BUILT/third_party/{BUILD*,*.bash}
 find docs/ -type f -name "*.md" -print0 \
   | xargs -n 1 -0 sed -i 's|(//|(https://github.com/enola-dev/enola/blob/main/|g'
 
+# We just always install https://github.com/marionebl/svg-term-cli, even if
+# we're then skipping it below, because this is quick, and doing it anyway helps
+# to detecting any CI build system regression with Nix etc.
+if ! [ -f "node_modules/.bin/svg-term" ]; then
+  npm install svg-term-cli
```

Comment:
I still think that 'npm ci' is the appropriate command to run if there's already package.json and lock file in the repo, otherwise those files are redundant.